test/phoenix/digester_test.exs

```diff
@@ -0,0 +1,34 @@
+defmodule Phoenix.DigesterTest do
+  use ExUnit.Case, async: true
+
+  test "fails when the given paths are invalid" do
+    assert {:error, :invalid_path} = Phoenix.Digester.compile("../some invalid?/path", "/other_invalid ?? /path")
+  end
+
+  test "digests and compress files" do
+    output_path = Path.join("tmp", "digested")
```

Comment:
We should use a different directory from the mix task. Just in case they are both running at the same file and something weird happens.